pkl-cli/pkl-cli.gradle.kts

```
@@ -218,39 +215,48 @@ fun Exec.configureExecutable(
  * Builds the pkl CLI for macOS/amd64.
  */
 val macExecutableAmd64: TaskProvider<Exec> by tasks.registering(Exec::class) {
-  dependsOn(":installGraalVmAmd64")
-  configureExecutable(
-    buildInfo.os.isMacOsX,
-    buildInfo.graalVmAmd64,
-    layout.buildDirectory.file("executable/pkl-macos-amd64")
-  )
+  if (buildInfo.os.isMacOsX) {
+    dependsOn(":installGraalVmAmd64")
+    configureExecutable(
+      buildInfo.graalVmAmd64,
+      layout.buildDirectory.file("executable/pkl-macos-amd64")
+    )
+  } else {
+    enabled = false
+  }
```

Shouldn't this just be
```suggestion
  enabled = buildInfo.os.isMacOsX
  dependsOn(":installGraalVmAmd64")
  configureExecutable(
    buildInfo.graalVmAmd64,
    layout.buildDirectory.file("executable/pkl-macos-amd64")
  )
```

? [throughout]

Surely, if `!enabled`, tasks this one `dependsOn` are not executed.